src/WebJobs.Script/Workers/Rpc/RpcWorkerProcess.cs

```diff
@@ -49,13 +51,15 @@ internal class RpcWorkerProcess : WorkerProcess
             _workerProcessArguments = rpcWorkerConfig.Arguments;
             _workerDirectory = rpcWorkerConfig.Description.WorkerDirectory;
             _hostingConfigOptions = hostingConfigOptions;
+            _environment = environment;
         }
 
         internal override Process CreateWorkerProcess()
         {
             var workerContext = new RpcWorkerContext(Guid.NewGuid().ToString(), RpcWorkerConstants.DefaultMaxMessageLengthBytes, _workerId, _workerProcessArguments, _scriptRootPath, _serverUri);
             workerContext.EnvironmentVariables.Add(WorkerConstants.FunctionsWorkerDirectorySettingName, _workerDirectory);
             workerContext.EnvironmentVariables.Add(WorkerConstants.FunctionsApplicationDirectorySettingName, _scriptRootPath);
+            workerContext.EnvironmentVariables.Add(RpcWorkerConstants.FunctionWorkerRuntimeVersionSettingName, _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeVersionSettingName));
```

Comment:
In prod, we were relying on this value being set at the machine-level by Antares. Because of this, I wasn't able to properly test. I'm setting this explicitly now for testing purposes (and will also be required when we eventually abstract the process-start away to enable cross-machine workers).